src/nat_cmd.c

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <modem/at_cmd.h>
+#include <modem/lte_lc.h>
+#include <net/socket.h>
+#include <shell/shell.h>
+#include <shell/shell_uart.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <zephyr.h>
+
+#include "nat_test.h"
+
+#define WAIT_TIME_S 3
+#define THREAD_STACK_SIZE 8192
+#define AT_LOG_TIMEOUT_S 20
```

Comment:
Are these defines used?

Reply:
Forgot about these when editing my commits. They belong to a future commit. 
Removed them.